docs-js/orchestration/chat-completion.mdx

```diff
@@ -136,6 +136,57 @@ const response = await orchestrationClient.chatCompletion({
 });
 ```
 
+### Prompt Registry
+
+You can define template content within the templating configuration, or refer to a template defined in the [Prompt Registry](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/prompt-registry).
```

Comment:
```suggestion
You can refer to a template defined in the [Prompt Registry](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/prompt-registry) instead of defining within the `templating` configuration.
```